src/components/Dashboard/index.jsx

```diff
@@ -11,15 +34,22 @@ class Dashboard extends React.Component {
   You will create a function renderUpselling that returns
   your Upselling jsx component and pass it as props.
    */
-  renderUpselling = () => (
-    <div>
-        upselling component
-    </div>
+
+  renderUpsellingStats = partnerStats => (
+    <ul>
+      { partnerStats.map(stat => (
+        <li key={stat.devNum}>
```

Comment:
dont use devNum as a key, this is not a unique key

Reply:
I had used devNum as the key because I was using dummy data however it makes more sense for the data to have an ID property.